syntax/neomake/qf.vim

```diff
@@ -0,0 +1,5 @@
+syntax clear
```

Comment:
The `let b:current_syntax = 'neomake_qf'` gets cleared with `syntax clear` though, so what is the point of setting it?
Should it get moved after `syntax clear`, or is it required for `syntax clear` somehow?